src/webhook/pkg/mutating_webhook/remove-resourcespec.go

```diff
@@ -26,18 +35,98 @@ func (a *PodMutator) mutatePod(pod *corev1.Pod) (*corev1.Pod, error) {
 		return mpod, nil
 	}
 
+	mpod = adjustResourceSpec(mpod)
+
+	return mpod, nil
+}
+
+// function to remove resource spec from the pod spec
+// add the cumulative resources as annotation to pod spec
+// add the peer-pod resource to the first container in the pod spec
+
+func adjustResourceSpec(pod *corev1.Pod) *corev1.Pod {
+
+	// Get total CPU resource requests
+	cpuRequest := utils.GetResourceRequestQuantity(pod, corev1.ResourceCPU)
+
+	// Get total CPU resource limits
+	cpuLimit := utils.GetResourceLimitQuantity(pod, corev1.ResourceCPU)
+
+	// Get total Memory resource requests
+	memoryRequest := utils.GetResourceRequestQuantity(pod, corev1.ResourceMemory)
+
+	// Get total Memory resource limits
+	memoryLimit := utils.GetResourceLimitQuantity(pod, corev1.ResourceMemory)
+
+	// Get total GPU resource requests
+	gpuRequest := utils.GetResourceRequestQuantity(pod, corev1.ResourceName(GPU_RESOURCE_NAME))
+
+	// log the resource values
+	logger.Printf("CPU Request: %s, CPU Limit: %s, Memory Request: %s, Memory Limit: %s, GPU Request: %s",
+		cpuRequest.String(), cpuLimit.String(), memoryRequest.String(), memoryLimit.String(), gpuRequest.String())
+
+	logger.Printf("Memory: %s", memoryRequest.String())
```

Comment:
Do we need this log statement, when it is in the one above, or have I missed something?

Reply:
No.... It's stale